pid_error:
  MIN_DISTANCE: 0.1
  MAX_DISTANCE: 30.0
  DESIRED_DISTANCE: .75

  CONTROL_DELAY_ESTIMATE: 0.1
  minLookDistance: 5 
  lookDistanceMultiplier: 0

  currentMode: "right"
  currentEnumMode: 2
  currentGapAngle: 0

control:
  KP: 0.1
  KI: 0.00
  KD: 0.001
  MAX_VEL: 5
  MIN_VEL: 2
  MAX_STEERING_ANGLE: 20
  N: 1
  K: 0.5

instruction:
  goodGapThresh: .4
  newInstructionGapCount: 1
  defaultInstruction: C
  instructionDirectory: '../explicit_instructions/'
  instructionFile: 'levine_instructions.dat'
  
